File: C/var.c
#include <stdio.h>

int main(void){
	int porta = 80;
	char ip[] = "192.168.0.1";
	float ver = 1.1;

	printf("MR Ronca\n");
	printf("Scan Porta: %d\n", porta);
	printf("Scan IP: %s\n", ip);
	printf("Scan version: %.1f\n", ver);

	return 0;
}

File: Python/var.py
#!/usr/bin/python
ip = "192.168.1.1"
port = 80
ver = 1.1
booleanTest = True

print("Scan version", ver)
print("Scanning Host:", ip, "on port", port)

print("IP -", type(ip))
print("Port -", type(port))
print("Version -", type(ver))
print("Bool -", type(booleanTest))

print("Scan version %.1f" %ver)
print("Scanning Host: %s on port: %d" %(ip,port))
print("This ticket are %r" %booleanTest)
